### PR TITLE
MauiBlazorWeb for 9.0

### PR DESCRIPTION
**Pull request by guardrex**
### MauiBlazorWeb 9.0 (second attempt 🙈😄) 

Based on the latest template with `InteractiveRenderSettings` added. After this is merged, I'll check the article to see if it requires updates to match the sample app changes.

I used the same local branch and didn't squash, so no worries on the old commits.

Seems to run well ...

image

### CODE REVIEW

File: 9.0/MauiBlazorWeb/MauiBlazorWeb.Shared/_Imports.razor [hunk 1]
```diff
@@ -0,0 +1,9 @@
+@using System.Net.Http
+@using System.Net.Http.Json
+@using Microsoft.AspNetCore.Components.Forms
+@using Microsoft.AspNetCore.Components.Routing
+@using Microsoft.AspNetCore.Components.Web
+@using static Microsoft.AspNetCore.Components.Web.RenderMode
```
Comment (BethMassi) [inline 9.0/MauiBlazorWeb/MauiBlazorWeb.Shared/_Imports.razor:6]:
```suggestion
@using static InteractiveRenderSettings
```

**Review by BethMassi — CHANGES_REQUESTED** [review]
You need to import the InteractiveRenderSettings class in the Share project for this to work with per page/component.

File: 9.0/MauiBlazorWeb/MauiBlazorWeb.Shared/_Imports.razor (hunk 1 above)
Comment (guardrex) [reply 9.0/MauiBlazorWeb/MauiBlazorWeb.Shared/_Imports.razor:6]: `@using static InteractiveRenderSettings` is already on Line 9 of the file.

I'll remove Line 6.

**Review by BethMassi — APPROVED** [review]
Looks good, thank you!

**Comment by guardrex** [comment]
I'll work on the article next and ping u on that PR when it goes up.

**Pull request merged**